hash/mac_hdflop.xml

```diff
@@ -5,125 +5,3347 @@ license:CC0
 -->
 <softwarelist name="mac_hdflop" description="Macintosh High Density Disk images">
 
+<!-- Operating system disks -->
+
+	<software name="htalk71">
+		<description>HangulTalk 7.1</description>
```

Comment:
Is this officially called “HangulTalk” in Latin script?  If not, please use RR transliteration “Hangeul”.

Reply:
Yes, it's officially written like that in Latin script. See: https://imgur.com/1BeL2Cb